examples/GrpcCli/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+# Script that exercises all the commands of the CLI.
+# Used to be executed as SDK "unit" tests.
```

Comment:
I think you can skip this historic reference

Reply:
Will do.